output/plantuml/plantuml_test.go

```diff
@@ -64,6 +94,34 @@ func TestOutputTable(t *testing.T) {
 	}
 }
 
+func TestOutputTableTemplate(t *testing.T) {
+	s := newTestSchema()
+	c, err := config.New()
+	if err != nil {
+		t.Error(err)
+	}
+	err = c.LoadConfigFile(filepath.Join(testdataDir(), "out_templates_test_tbls.yml"))
+	if err != nil {
+		t.Error(err)
+	}
+	// use the templates in the testdata directory
+	c.Templates.PUML.Table = path.Join(testdataDir(), c.Templates.PUML.Table)
```

Comment:
I think you would better use the `filepath` package.

```suggestion
	c.Templates.PUML.Table = filepath.Join(testdataDir(), c.Templates.PUML.Table)
```

Reply:
Yes, will do :slightly_smiling_face: